src/draw.c

```diff
@@ -554,4 +554,12 @@ void draw(void)
         cairo_surface_destroy(image_surface);
         r_free_layouts(layouts);
 }
+
+void draw_free(void)
```

Comment:
rebase?

Reply:
Oh, I have to clarify myself: I would use `*_free(...);` only for structures, which are dynamic elements. I would rename it to `_deinit(...`, because it deallocates only static elements.